docs/source/contributing/dev-contributions/contrib_guide.rst

```diff
@@ -120,7 +120,9 @@ including partial or in-progress work, is appreciated.
 
 Contribution Workflow
 ---------------------
+[Link to full alt-text of Contribution Workflow flow chart.](https://gist.githubusercontent.com/MarsBarLee/2dc5e9c83adf0555d08277b2262302e5/raw/be6a0c46f58d7135a75df7747698cef676b9cdc0/Jupyter%2520Documentation:%2520Contribution%2520Workflow%2520Alt-Text)
```

Comment:
For this alt-text, I provided both short alt-text and long alt-text. In this context, longer alt-text is valuable as short-alt text would be excluding essential information in this flowchart. I have linked out to the long alt-text.

During the workshop, I linked to a personal gist due to the ease of use and speed. However, for submission into the Jupyter documentation, I think linking to a markdown file in the Jupyter documentation would be better. 

Where would a markdown file with an image's long text be contained? In this case, the file is in contributing/dev-contributions but the image is in _static/_images. I'm leaning towards putting it in the images folder. The markdown file could be titled 'contribution_workflow_long_alt_text.md' so it would show next to the image 'contribution_workflow.png'.

In reference to [WCAG](https://www.w3.org/WAI/tutorials/images/complex/#a-text-link-to-the-long-description-adjacent-to-the-image) and [Deque](https://dequeuniversity.com/tips/long-descriptions), both refer to linking out to long alt-text, but do not advice where the text should be hosted.

I could create a separate PR after this PR submitted to change the link and add the long alt-text file.

Reply:
Hmmm - what if we provided this as a footnote? Here's the reStructuredText way to write footnotes in Sphinx: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes